File: Day 23/Aayushi-Mittal.py
# To count the total number of odd and even numbers in array.

n=int(input()) 
arr=[] 
even=0
odd=0
for i in range(0,n):
    arr.append(int(input()))
    if arr[i]%2==0:
        even++
    else:
        odd++
print("> No of even elements in the array : ",even)
print("> No of odd elements in the array : ",odd)

File: Day 23/Aayushi-Mittal.cpp
//To count the total number of odd and even numbers in array.

#include<iostream>
using namespace std;
int main()
{
  int n=10, arr[n], even=0, odd=0;
  cin>>n;
  for(int i=0; i<n; i++)
  {
    cin>>arr[i];
    if(arr[i]%2==0)
        even++;
    else
        odd++;
  }
  cout<<"> No of even elements in the array : "<<even<<endl;
  cout<<"> No of odd elements in the array : "<<odd<<endl;
}
